src/Microsoft.OpenApi/Models/OpenApiResponse.cs

```diff
@@ -231,6 +224,22 @@ public void SerializeAsV2WithoutReference(IOpenApiWriter writer)
                         writer.WriteEndObject();
                     }
 
+                    if (Content.Values.Any(m => m.Examples != null && m.Examples.Any()))
+                    {
+                        writer.WritePropertyName("x-examples");
```

Comment:
could we use a constant instead of defining it a couple of times all over the place?